sdk/identity/identity/src/credentials/usernamePasswordCredential.ts

```diff
@@ -42,7 +42,7 @@ export class UsernamePasswordCredential implements TokenCredential {
   ) {
     if (!tenantId || !clientId || !username || !password) {
       throw new Error(
-        "UsernamePasswordCredential: tenantId, clientId, username and password are required parameters."
+        "UsernamePasswordCredential: tenantId, clientId, username and password are required parameters. Visit https://aka.ms/azsdk/js/identity/usernamepasswordcredential/troubleshoot to troubleshoot this issue."
```

Comment:
Do these logs/errors match the ones in Java? I wonder what inconsistencies we might have.